recipes/heavydb/meta.yaml

```
@@ -0,0 +1,575 @@
+{% set version = "6.0.0" %}  # PEP 386
+{% set source_version = version %}
+{% set base_version = version %}
+{% set number = "4" %}
+{% set cuda_enabled = cuda_compiler_version is not undefined and cuda_compiler_version == '11.0' %}
```

Isn't this equivalent? 
```suggestion
{% set cuda_enabled = cuda_compiler_version == '11.0' %}
```

---

No. IIRC, if `cuda_compiler_version` is not defined then comparison `cuda_compiler_version == '11.0'` cannot be performed.